frontend/src/Components/loginForm.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import './login.css';
+
+export const LoginForm = () => {
+    const [username, setUsername] = useState<string>('');
+    const [password, setPassword] = useState<string>('');
+    
+    const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        try {
+          const response = await axios.post("http://localhost:8000/login/password", {
+            username,
+            password,
+          });
+          console.log(response.data)
+        } catch(error) {
+            console.error(error);
+        }
+    };
+    
+  const handleUsernameChange = (event: React.ChangeEvent<HTMLInputElement>, setUsername: React.Dispatch<React.SetStateAction<string>>) => {
+    setUsername(event.target.value);
+  };
+
+  const handlePasswordChange = (event: React.ChangeEvent<HTMLInputElement>, setPassword: React.Dispatch<React.SetStateAction<string>>) => {
```

Comment:
so you actually don't need to pass `setPassword` in as an argument here since `setPassword` is within the scope of the entire component and you can just access it implicitly

Reply:
Okay makes sense.